src/RecoveryServices/RecoveryServices.Backup.Autorest/examples/Get-AzRecoveryServicesBackupPolicy.md

```diff
@@ -1,22 +1,141 @@
-### Example 1: {{ Add title here }}
+### Example 1: Get all backup policies in a recovery services vault
 ```powershell
-{{ Add code here }}
-```
+$pol = Get-AzRecoveryServicesBackupProtectionPolicy -ResourceGroupName "myresourcegroup" -VaultName "myvault"
+$pol | fl 
```

Comment:
just give the output without fl. also hide the values like subscription ID and resourcegroups. Have generalized values like  xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx, resourcegroupname, policyName etc (this is to keep security and maintain production quality)